backend/ecs_tasks/delete_files/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Do we need Python 3.8 here? If we are moving to Python 3.8, can you upgrade the Local Development Guide, the Github Actions and the Docker image too?

Reply:
I noticed this too but just disregarded it as a change that must've happened in pip-compile, and I suppose I was running it under python 3.8.

Reply:
Ok let's ignore it for now then.